src/io/io.cpp

```diff
@@ -15,6 +15,9 @@
 
```

Comment:
you need to fix scons build system if you want to move this file.

Reply:
src/question_match/question_matcher.cpp
and this too.

Reply:
I know, just haven't done that yet.

Reply:
Fixed SCONS for linux, but haven't fixed java and Windows pipelines yet.